src/Datepicker/Datepicker.tsx

```diff
@@ -39,42 +47,24 @@ export const Datepicker: FC<DatepickerProps> = ({
   fromDate = new Date(),
```

Comment:
Were you planning to solve the timezone issue in signup? I'm wondering if might be better to set it by default to generate UK time only start dates

Reply:
Yeah, planning to fix the time zone issue on a separate PR! Will open the PR this morning